api/src/queries/occurrence/template-observation-queries.ts

```diff
@@ -22,6 +22,7 @@ export const getTemplateObservationsSQL = (surveyId: number): SQLStatement | nul
     os.update_date,
     os.create_date,
     os.key,
+    os.file_name,
```

Comment:
I think this `getTemplateObservationsSQL` function is a duplicate of `getLatestSurveyOccurrenceSubmission`?

Reply:
very close ... I removed the getTemplateObservationSQL and added the pieces I needed to the other function.